SU-HSDO/suhumsci
# SHS-151: Update Decanter Library and theme tooling to Node18

# READY FOR REVIEW

## Summary
- Updates decanter library to the latest version in anticipation that a new release with font performance improvements is coming shortly.
- Update theme compiling and dependencies to use Node18

## Urgency
low

## Steps to Test
1. Go to the the `humsci_basic` directory and run `nvm use` then `npm run start` - Note that there are a lot of deprecation warnings due to decanter but the theme's should still compile.
2. Make various CSS and JS change and verify the development process still works despite the deprecation warnings.
3. Review the Percy link and verify there are no style changes associated with the updates: 
https://percy.io/da667d23/suhumsci/builds/25074687
https://percy.io/da667d23/suhumsci/builds/25074677

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
